Bottom: f9c37fd54ce593907a69c5885324347e0802844a
Top:    ea1fde9dd4827e5c7a089c41ea14b3b172ce5b83
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 18:45:59 +0000

Refresh of add-missing-return-value-in-ac

---

diff --git a/src/action.c b/src/action.c
index 8d5931a..ba9ca44 100644
--- a/src/action.c
+++ b/src/action.c
@@ -7095,6 +7095,7 @@ ActionElementList (int argc, char **argv, int x, int y)
   if (!e)
     {
       int nx, ny, d;
+      char *as;
 
 #ifdef DEBUG
       printf("  ... Footprint not on board, need to add it.\n");
@@ -7604,6 +7605,7 @@ ActionImport (int argc, char **argv, int x, int y)
   if (mode && strcasecmp (mode, "setdisperse") == 0)
     {
       const char *ds, *units;
+      int d, r;
       char buf[50];
 
       ds = ARG (1);
